zephyr/esp32/CMakeLists.txt

```diff
@@ -187,7 +187,7 @@ if(CONFIG_SOC_SERIES_ESP32 OR CONFIG_SOC_SERIES_ESP32_NET)
 
   endif()
 
-  if (CONFIG_MCUBOOT)
+  if (USE_ESP32_HAL_MCUBOOT)
```

Comment:
is this strictly for mcuboot?

Reply:
Yes, it is. Should I keep `CONFIG_MCUBOOT`?

Reply:
it's ok, just was surprised HAL code specific to a particular bootloader. @nordicjm

Reply:
@sylvioalves can you check mcuboot please? There is an issue with the espressif build on CI, RAM overflows by 5 bytes, so it is likely that this will be a problem for anyone wanting to use mcuboot

Reply:
@nordicjm, can you point the CI issue? I'll check right away.

Reply:
See CI results on https://github.com/mcu-tools/mcuboot/pull/1774, thank you